### PR TITLE
feat(obj-c): add cash app methods (EIT-4059)

### PR DESCRIPTION
**Pull request by ScottAntonacAP**
## Summary of Changes

This PR exposes methods and classes relating to Cash App for Obj C. These include:

- `validateCashAppOrder()`
- `signCashAppOrderToken()`
- `cashAppClientId`

### CODE REVIEW

File: Sources/Afterpay/Wrappers/ObjcWrapper.swift [hunk 1]
```diff
@@ -168,6 +170,370 @@ public final class ObjcWrapper: NSObject {
     }
   }
 
-  // swiftlint:enable function_parameter_count
+  // Cash App
+  // swiftlint:disable type_name
+
+  @objc static var cashAppClientId: String {
+    Afterpay.cashAppClientId ?? ""
```
Comment (mmroz) [inline Sources/Afterpay/Wrappers/ObjcWrapper.swift:177]: Any reason not to make this optional?

File: Sources/Afterpay/Wrappers/ObjcWrapper.swift [hunk 2]
```diff
@@ -168,6 +170,370 @@ public final class ObjcWrapper: NSObject {
...
+    }
+
+    static func error(error: Error) -> CashAppValidationFailedReasonError {
+      CashAppValidationFailedReasonError(error)
+    }
+  }
+
+  @objc(APCashAppValidationFailedReasonNilData)
+  public class CashAppValidationFailedReasonNilData: CashAppValidationFailedReason {
+    init(_ void: ()) {}
```
Comment (mmroz) [inline Sources/Afterpay/Wrappers/ObjcWrapper.swift:447]: Why do these take a void?

**Review by mmroz — APPROVED** [review]
LGTM just a couple of questions

File: Sources/Afterpay/Wrappers/ObjcWrapper.swift (hunk 1 above)
Comment (ScottAntonacAP) [reply Sources/Afterpay/Wrappers/ObjcWrapper.swift]: no reason - great pickup. addressed in [71f61b6](https://github.com/afterpay/sdk-ios/pull/262/commits/71f61b64c75668143e62c26294ed233250623d8a)

File: Sources/Afterpay/Wrappers/ObjcWrapper.swift (hunk 2 above)
Comment (ScottAntonacAP) [reply Sources/Afterpay/Wrappers/ObjcWrapper.swift:447]: no reason other than to keep consistency with the existing obj-c wrapper code.

**Pull request merged**